pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -2,20 +2,23 @@ package bootstrappolicy
 
 import (
 	"reflect"
+	"sort"
 	"testing"
 
 	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 
 	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	scc "github.com/openshift/origin/pkg/security/securitycontextconstraints"
 )
 
 func TestBootstrappedConstraints(t *testing.T) {
-	expectedConstraints := []string{
+	// ordering of expectedConstraintNames is important, we check it against scc.ByPriority
+	expectedConstraintNames := []string{
 		SecurityContextConstraintPrivileged,
-		SecurityContextConstraintRestricted,
 		SecurityContextConstraintNonRoot,
 		SecurityContextConstraintHostMountAndAnyUID,
 		SecurityContextConstraintHostNS,
+		SecurityContextConstraintRestricted,
 		SecurityContextConstraintsAnyUID,
```

Comment:
Is this order correct? Why `anyuid` isn't the last? It has `priority: 10` AFAIR. What I'm missing?